### PR TITLE
add FuzzyCraftingTable

### PR DESCRIPTION
**Pull request by cybcaoyibo**
Imgur

### CODE REVIEW

**Comment by SinusoidalC** [comment]
sweetI Is it match fuzzy crafting upgrade?

**Comment by bziemons** [comment]
Needs to be tested well, otherwise it looks good

File: common/logisticspipes/blocks/crafting/LogisticsCraftingTableTileEntity.java [hunk 1]
```diff
@@ -145,6 +162,16 @@ public void readFromNBT(NBTTagCompound par1nbtTagCompound) {
 		inv.readFromNBT(par1nbtTagCompound, "inv");
 		matrix.readFromNBT(par1nbtTagCompound, "matrix");
 		placedBy = par1nbtTagCompound.getString("placedBy");
+		NBTTagList lst = par1nbtTagCompound.getTagList("fuzzyFlags");
+		if(lst != null) {
```
Comment (davboecki) [inline common/logisticspipes/blocks/crafting/LogisticsCraftingTableTileEntity.java]: Checking for null doesn't work here becuase an empty list is returned when the tag is not found.
Use hasKey on the NBTCompund.

**Comment by davboecki** [comment]
Other than that, everything else looks good. FIrst small test didn't show any problems.

**Pull request merged**